src/Blazored.LocalStorage/BrowserStorageProvider.cs

```diff
@@ -73,6 +77,15 @@ public void RemoveItem(string key)
             _jSInProcessRuntime.InvokeVoidAsync("localStorage.removeItem", key);
         }
 
+        public void RemoveItems(List<string> keys)
+        {
+            CheckForInProcessRuntime();
+            foreach (var key in keys)
+            {
+                _jSInProcessRuntime.InvokeVoidAsync("localStorage.removeItem", key);
```

Comment:
This call should use the non-async method
```suggestion
                _jSInProcessRuntime.InvokeVoid("localStorage.removeItem", key);
```

Reply:
You're right - I completely dropped the ball on keeping the APIs synchronized. Committing them now, tests pass but would you mind taking a look I'm on the right track here?